### PR TITLE
PXB-2815 link for error messages (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
A  source/em/instant.rst
This doc is used only for a link when the user sees a specific error message.

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+MySQL release 8.0.29 extended the support for ALTER TABLE … ALGORITHM=INSTANT to allow users to add columns in any position of the table or drop columns. As part of this work, the redo log format has changed for all DML operations on the server. This new redo log format has a design flaw that can cause data corruption for tables with INSTANT ADD/DROP COLUMNS.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:1]:
```suggestion
MySQL release 8.0.29 extended the support for `ALTER TABLE … ALGORITHM=INSTANT` to allow users to add columns in any position of the table or drop columns. As part of this work, the redo log format has changed for all DML operations on the server. This new redo log format has a design flaw that can cause data corruption for tables with INSTANT ADD/DROP COLUMNS.
```

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+The corruption happens when the crash recovery takes place. InnoDB applies redo logs at startup. Xtrabackup copies the redo log during backup and applies it as part of the --prepare step to bring the backup to a consistent state.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:3]:
```suggestion
The corruption happens when the crash recovery takes place. InnoDB applies redo logs at startup. Xtrabackup copies the redo log during backup and applies it as part of the `--prepare` step to bring the backup to a consistent state.
```

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Percona fixed the corruption issue and several other issues with the INSTANT ADD/DROP column feature in the upcoming Percona Server for MySQL 8.0.29 (check PS-8291 /  PS-8292 / PS-8303 for more details). Percona XtraBackup 8.0.29 will be able to take backups of Percona Server for MySQL 8.0.29 with tables that have INSTANT ADD/DROP COLUMNS. However, the current version of MySQL 8.0.29 still has this flaw, making it unsafe to take backups.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:5]:
```suggestion
Percona fixed the corruption issue and several other issues with the INSTANT ADD/DROP column feature in the upcoming Percona Server for MySQL 8.0.29 (check [PS-8291](https://jira.percona.com/browse/PS-8291) /  [PS-8292](https://jira.percona.com/browse/PS-8292) / [PS-8303](https://jira.percona.com/browse/PS-8303) for more details). Percona XtraBackup 8.0.29 will be able to take backups of Percona Server for MySQL 8.0.29 with tables that have INSTANT ADD/DROP COLUMNS. However, the current version of MySQL 8.0.29 still has this flaw, making it unsafe to take backups.
```

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+mysql> SELECT NAME FROM INFORMATION_SCHEMA.INNODB_TABLES WHERE TOTAL_ROW_VERSIONS > 0;
++---------+
+| NAME    |
++---------+
+| test/t1 |
+| test/t2 |
+| test/t3 |
++---------+
+3 rows in set (0.02 sec)
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:23]: please add it as formatted code.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+If this query shows an empty result set, you are all good. Percona Xtrabackup will take backups of your MySQL 8.0.29 servers. If not, please run OPTIMIZE TABLE on the list of tables before taking backup.
+
+
+Percona XtraBackup Error Message
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:29]:
```suggestion
# Percona XtraBackup Error Message
```

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+If Percona XtraBackup detects that MySQL 8.0.29 server has tables with instant add/drop columns, it aborts with the following error message
+
+2022-07-01T15:18:35.127689+05:30 0 [ERROR] [MY-011825] [Xtrabackup] Found tables with row versions due to INSTANT ADD/DROP columns
+2022-07-01T15:18:35.127714+05:30 0 [ERROR] [MY-011825] [Xtrabackup] This feature is not stable and will cause backup corruption.
+2022-07-01T15:18:35.127723+05:30 0 [ERROR] [MY-011825] [Xtrabackup] Tables found:
+2022-07-01T15:18:35.127730+05:30 0 [ERROR] [MY-011825] [Xtrabackup] test/t1
+2022-07-01T15:18:35.127737+05:30 0 [ERROR] [MY-011825] [Xtrabackup] test/t2
+2022-07-01T15:18:35.127744+05:30 0 [ERROR] [MY-011825] [Xtrabackup] test/t3
+2022-07-01T15:18:35.127752+05:30 0 [ERROR] [MY-011825] [Xtrabackup] Please run OPTIMIZE TABLE or ALTER TABLE ALGORITHM=COPY on all listed tables to fix this issue.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:39]: please add it as formatted code.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+Summary
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:41]:
```suggestion
# Summary
```

File: storage/innobase/xtrabackup/doc/source/em/instant.rst [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+Algorithm INSTANT is the new default in 8.0.29. If you do not specify an algorithm, all ALTER TABLE ADD/DROP COLUMN statements will use the default algorithm.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/em/instant.rst:43]:
```suggestion
Algorithm INSTANT is the new default in 8.0.29. If you do not specify an algorithm, all `ALTER TABLE ADD/DROP COLUMN` statements will use the default algorithm. 
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:1]: rewritten

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:3]: Added.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 3 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:5]: Rewritten and added.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 4 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:23]: Done.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 5 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:29]: Added as title

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 6 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:39]: Done.

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 7 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:41]: Made a title

File: storage/innobase/xtrabackup/doc/source/em/instant.rst (hunk 8 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/em/instant.rst:43]: Added.

**Pull request closed without merging**